source/modules/MEI.visual.xml

```diff
@@ -679,6 +679,18 @@
           <rng:ref name="data.BOOLEAN"/>
         </datatype>
       </attDef>
+      <attDef ident="opening.vertical" usage="opt">
```

Comment:
Would it make sense to move this below  "opening" attDef? In general, do we follow alphabetical order for attDefs?

Reply:
We can. I left it after `@angle.optimize` because they it relates to it. I am not sure we follow alphabetical order systematically.

Reply:
Ok, fine with that